packages/jaeger-ui/src/utils/prefix-url.tsx

```diff
@@ -15,6 +15,18 @@
 import sitePrefix from '../site-prefix';
 
 const origin = process.env.NODE_ENV === 'test' ? global.location.origin : window.location.origin;
+
+/**
+ * Escape any RegEx characters in a given string
+ *
+ * @param  {string} value The string which needs to be escaped
+ * @return {string} Escaped RegEx string
+ */
+function regExEscape(value?: string) {
```

Comment:
this function will always be called with a `string`, so the `?` and the cast on the next line are unnecessary.
But since this is only called once, you could simply take the `replace` on line 27 and use it in the falsy branch of the ternary on line 41, as such:
```
const s = sitePref == null ? '' : sitePref.replace(/[-/\\^$*+?.()|[\]{}]/g, '\\$&');
```
with a comment above saying the `replace` will escape any RegEx characters in the `sitePref`